### DISCUSSION TITLE
stoicism/about/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#7] [General]
# stoicism/about/

https://shanenull.com/stoicism/about/

**Comment by shane0** [comment]
- a no nonsense & powerful workflow
- project templates: [cookiecutter](https://github.com/cookiecutter/cookiecutter)
- task tracking: [todocli](https://github.com/francoischalifour/todo-cli)
- modular command line automation: [click](https://click.palletsprojects.com/en/8.1.x/)
- powerful documentaiton: [mkdocs](https://squidfunk.github.io/mkdocs-material/)
- measurable readability: [lexi](https://github.com/Rebilly/lexi)
